Platform/ArrowlakeBoardPkg/Script/StitchIfwiConfig_arls.py

```diff
@@ -147,9 +145,6 @@ def get_xml_change_list (platform, plt_params_list):
         # Path                                                                      | value |
         # =========================================================================================
         #   Region Order
-        ('./FlashSettings/BiosConfiguration/PchTopSwapOverride',                                    '4MB'),
-        ('./FlashSettings/BiosConfiguration/SocTopSwapOverride',                                    '4MB'),
-        ('./FlashSettings/BiosConfiguration/BiosRedAssistance',                                     'Disabled'),
```

Comment:
This change will impact SBL resiliency feature. When “BiosRedAssistance” is enabled, CSME takes care of TS switching for corruptions in ucode/ACM/IBB. Better update this in a separate PR.